scripts/leaf-scripts/update_data_metadata.php

```diff
@@ -0,0 +1,134 @@
+<?php

```

Comment:
Please rename to update_devenv_data_metadata.php or similar.

This is to improve the distinction between a script meant to be run in development or prod environments, and reduce potential mishaps on the command line.